sunflower/gui/preferences/associations.py

```diff
@@ -1,6 +1,6 @@
 from gi.repository import Gtk
-from gui.input_dialog import InputDialog, ApplicationInputDialog
-from widgets.settings_page import SettingsPage
+from ..input_dialog import InputDialog, ApplicationInputDialog
+from sunflower.widgets.settings_page import SettingsPage
```

Comment:
Why import one from `sunflower.widgets` and other from `..input_dialog`? It needs to be consistent.

Reply:
Fair. My thought process was that `from ...widgets.settings_pane import SettingsPane`, with three `.`, is getting a bit hard to understand, so I made all third-level imports absolute. Happy to change that of course.